src/lib/fetchAPI.js

```diff
@@ -44,9 +44,8 @@ export async function requestEventList() {
   return data;
 }
 
-export async function requestEvent(id) {
-  const token = process.env["ADMIN_TOKEN"];
-  const res = await fetch(`https://eventmakers-api.fly.dev/events/${id}`, {
+export async function requestEvent(token, eventid) {
+  const res = await fetch(`https://eventmakers-api.fly.dev/events/${eventid}`, {
     method: "GET",
     headers: {
       Authorization: `Bearer ${token}`,
```

Comment:
I dont think we need to pass the token for this URL

See image

![image](https://github.com/alfonsusac/devscale-eventmaker/assets/20208219/9b7c33f7-7e46-4705-bd6b-c2fb8432b5fb)